README.md

```diff
@@ -8,19 +8,19 @@ Our goal is to build a generalizable monitoring and evaluation (M&E) platform th
 
 ## Schema
 
-[<img src='./docs/Labs37 DB Schema.png'/> DB Designer Schema Link](https://dbdesigner.page.link/KdYw5jrqdVDgWU3a8)
+[<img src='./docs/Labs37 DB Schema.png'/> DB Designer Schema Link](https://dbdesigner.page.link/VWjy5xWKghDumgmV8)
```

Comment:
Thanks for fixing that datatype error! `location_type_id` is definitely an integer, not varchar(255).